tests/saitests/py3/sai_qos_tests.py

```diff
@@ -4648,7 +4657,7 @@ def runTest(self):
                                               packet_size=packet_size)
 
             # Send packets short of triggering pfc
-            send_packet(self, src_port_id, pkt, pkts_num_trig_pfc)
+            send_packet(self, src_port_id, pkt, pkts_num_trig_pfc // cell_occupancy - 1 - pkts_num_margin)
```

Comment:
Why do we need this change? What happened if `cell_occupance == 1, pkts_num_margin=2`?

Reply:
Hi @bingwang-ms , for Nvidia platforms, we need larger packet size for the xoff test, otherwise the descriptor will be exhausted and packets will be dropped. 
We use packet_size=300 in the QoS SAI PFCTest(Defined in https://github.com/sonic-net/sonic-mgmt/blob/master/tests/qos/files/qos.yml#L24-L43) , so we can just inherit the value to the PCBBPFCTest test. 
In this case, since our cell size is 144 for SPC1/2/3, we need this change in line 4660 to calculate the correct packet number, and this is also how we do in the QoS SAI PFCTest(https://github.com/sonic-net/sonic-mgmt/blob/master/tests/saitests/py3/sai_qos_tests.py#L1084-L1085)